tests/quantization/ggml/test_ggml.py

```diff
@@ -171,6 +173,19 @@ def test_qwen2_q4_0(self):
         EXPECTED_TEXT = "Hello.jsoup\n\nI am a beginner"
         self.assertEqual(tokenizer.decode(out[0], skip_special_tokens=True), EXPECTED_TEXT)
 
+    def test_llama3_q4_0(self):
```

Comment:
The string that you are testing could be improved! Specifically to take into account the special tokens and etc. 
But alright otherwise!